lib/postgrex/types.ex

```diff
@@ -292,10 +292,12 @@ defmodule Postgrex.Types do
 
   defp fetch!(table, oid) do
     case :ets.lookup(table, oid) do
+      [{_, info, nil}] ->
+        raise ArgumentError, "no extension found for oid `#{oid}`: " <> inspect(info)
       [value] ->
         value
       [] ->
-        raise ArgumentError, message: "no extension found for oid `#{oid}`"
+        raise ArgumentError, "no extension found for oid `#{oid}`"
```

Comment:
Ah yes, thank you, I didn't check the SQL, I though this was handling in `find_extension/4` because it has the `nil` clause at the end. I saw I had gained ~100 types but turns out these are for array etc that we cant handle.